src/log.h

```diff
@@ -89,4 +105,9 @@ class LogStream
 // Note: LogType::DEBUG will prepend __FILE__ and __LINE__ to the debug message
 
 #define LOG(...) LogStream(__FILE__, __LINE__, LogType::__VA_ARGS__)
+
+#define DISABLE_LOG(type) bpftrace::Log::get().disable(LogType::type)
+#define ENABLE_LOG(type) bpftrace::Log::get().enable(LogType::type)
+#define LOG_ENABLED(type) bpftrace::Log::get().is_enabled(LogType::type)
```

Comment:
Let's add these as needed. I don't see any callers yet. I also can't see a reason why callers would need this.
```suggestion
```